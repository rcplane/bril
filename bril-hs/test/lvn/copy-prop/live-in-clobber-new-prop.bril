@main {
  a: int = const 0;
.lbl:
  b: int = id a;
  c: int = id b;
  d: int = id c;
  a: int = const 1;
  # b should be replaced with d
  print b;
}
