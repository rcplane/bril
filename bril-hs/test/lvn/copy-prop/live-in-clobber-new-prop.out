@main {
  a: int = const 0;
.lbl:
  d: int = id a;
  a: int = const 1;
  print d;
}
